httpapi/api.go

```diff
@@ -0,0 +1,19 @@
+package httpapi
+
+import (
+	"log"
+
+	"github.com/gin-gonic/gin"
+	"github.com/nibble-4bits/ondemand-go-bootcamp/usecase"
+)
+
+func StartServer(service usecase.PokemonService) {
```

Comment:
Could you please elaborate on this comment? 😅

Reply:
Whoops, sorry again, my brain cables crossed over and I got confused (I was kind on a hurry when I was doing the review 😢).

Let me correct the comment in this case:
Instead of having a concrete type as a parameter of your StartServer function, it would be better to receive an interface. In the future, this change will make the task of unit testing this function less painful, as right now you will need an actual implementation to test it. By using an interface you can use mocks to unit test this.

Reply:
Oh I get it now. I think in this case it's fine, right? Because`PokemonService` is an interface already